ui/options_network.ui

```diff
@@ -228,6 +228,47 @@
      </layout>
     </widget>
    </item>
+   <item>
+    <layout class="QHBoxLayout" name="horizontalLayout_3">
+     <item>
+      <widget class="QLabel" name="label_3">
+       <property name="text">
+        <string>Request timeout:</string>
```

Comment:
For translations, I think it would be better to have `Request timeout in seconds:` instead of 2 labels.

Reply:
Changed that